### PR TITLE
Koji build follow-up: correct url, state, target and more logging

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
- Fix `state` and `url` for build status.
- Pick the correct target.
- Add missing test file. (Forgot to add when restructuring the commits in the first PR.)

### CODE REVIEW

File: packit_service/worker/build/koji_build.py [hunk 1]
```diff
@@ -136,12 +148,21 @@ def run_build(
...
-        )
+        try:
+            out = self.api.up.koji_build(
+                scratch=self.is_scratch,
+                nowait=True,
+                koji_target=target,
+                srpm_path=self.srpm_path,
+            )
+        except PackitCommandFailedError as ex:
+            logger.warning(
```
Comment (csomh) [inline packit_service/worker/build/koji_build.py:159]: Wouldn't make more sense to log an error if we raise an error?

Comment (lachmanfrantisek) [reply packit_service/worker/build/koji_build.py:159]: This code is located in packit -- we already do it but without the stout/stderr.

(I'll take a look at it.)

**Comment by lachmanfrantisek** [comment]
I've also fixed the used target and added a test for it. Please, PTAL

File: packit_service/worker/build/koji_build.py (hunk 1 above)
Comment (lachmanfrantisek) [reply packit_service/worker/build/koji_build.py:159]: OK, I understood your comment as why we don't do this in a place the original error is raised.

**Pull request merged**